Deprecate rcParams["datapath"] in favor of mpl.get_data_path(). (#16417)
matplotlib/matplotlib: doc/api/next_api_changes/removals.rst

- The ``pgf.debug``, ``verbose.fileo`` and ``verbose.verbose.level`` rcParams,
  which had no effect, have been removed.
- Support for setting :rc:`mathtext.default` to "circled" has been removed.

> - The ``datapath`` rcParam has been removed.  Use `.get_data_path` instead.

Reviewer: ```suggestion
- The ``datapath`` rcParam is ignored.  Use `.get_data_path` instead.
```
